applications/HDF5Application/python_scripts/hdf5_io.py

```diff
@@ -244,13 +246,21 @@ def ExecuteFinalizeSolutionStep(self):
         delta_time = self._model_part.ProcessInfo[KratosMultiphysics.DELTA_TIME]
         self._output_time += delta_time
         self._output_step += 1
+
         if self._output_time >= self._output_time_frequency or self._output_step == self._output_step_frequency:
             hdf5_file = self._hdf5_file_factory.Open(self._get_current_file_name())
             for output in self._list_of_outputs:
                 output.Execute(self._model_part, hdf5_file)
             self._output_time = 0.0
             self._output_step = 0
 
+            del hdf5_file # force closing of file
```

Comment:
Closing the file is currently implemented in the destructor of the hdf-file
Alternatively we could expose this